sdks/python/container/boot.go

```diff
@@ -170,6 +176,28 @@ func main() {
 	log.Fatalf("Python exited: %v", execx.Execute("python", args...))
 }
 
+// setup wheel specs according to installed python version
+func setupAcceptableWheelSpecs() error {
+	cmd := exec.Command("python", "-V")
+	stdoutStderr, err := cmd.CombinedOutput()
+	if err != nil {
+		return err
+	}
+	re := regexp.MustCompile(`Python (\d)\.(\d).*`)
+	pyVersions := re.FindStringSubmatch(string(stdoutStderr[:]))
+	if len(pyVersions) != 3 {
+		return fmt.Errorf("cannot get python version with expected format")
```

Comment:
How about we say "Cannot get parse Python version from ..." and append the output?

Reply:
It would be better!